Tests/TuistGeneratorTests/ProjectMappers/ResourcesProjectMapperTests.swift

```diff
@@ -428,7 +428,10 @@ final class ResourcesProjectMapperTests: TuistUnitTestCase {
         // Given
         let projectName = "sdk-with-dash"
         let targetName = "target-with-dash"
-        let expectedBundleName = "\(projectName)_\(targetName.replacingOccurrences(of: "-", with: "_"))"
+        let sanitizedTargetName = targetName
+            .replacingOccurrences(of: "-", with: "_")
+            .replacingOccurrences(of: "/", with: "_")
+        let expectedBundleName = "\(projectName)_\(sanitizedTargetName)"
```

Comment:
I would probably hardcode this. The less business logic in tests the better.
```suggestion
        let expectedBundleName = "sdk_with_dash_target_with_dash"
```